src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -713,10 +713,15 @@ public enum NuGetLogCode
         NU5040 = 5040,
 
         /// <summary>
-        /// Error_ReadmeCannotOpenFile
+        /// Error_ProjectJsonPack_Deprecated_And_Disabled
         /// </summary>
         NU5041 = 5041,
```

Comment:
Are you changing this error semantics/meaning?

Reply:
Oh no, I mistakenly pasted the description there.

I'll remove it. Nice catch.